clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/util/id/converter/NoOpDcosAtomicOperationConverter.groovy

```diff
@@ -0,0 +1,26 @@
+package com.netflix.spinnaker.clouddriver.dcos.deploy.converter
+
+import com.netflix.spinnaker.clouddriver.dcos.DcosOperation
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperation
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperations
+import com.netflix.spinnaker.clouddriver.security.AbstractAtomicOperationsCredentialsSupport
+import org.springframework.stereotype.Component
+
+/**
+ * Adding this for initial pull request to satisfy FeatureController requirement of at least
+ * one atomic operation enabled when starting clouddriver with just dcos enabled.
+ */
+@DcosOperation(AtomicOperations.DELETE_SECURITY_GROUP)
+@Component("noopOracleBMCSDescription")
+class NoOpDcosAtomicOperationConverter extends AbstractAtomicOperationsCredentialsSupport {
```

Comment:
You should not need to define a NoOp operation now. Clouddriver should just emit a warning on startup that it didn't discover any operations. But it should start up just fine.